### ISSUE TITLE
Rename "termination" criterion to "continuation" criterion

### ISSUE THREAD

**Issue by lamblin** [lisa-lab/pylearn2#1405]
This better matches the use of `Or` and `And`.
See #720 and #1311 for the initial reports.
We should remove the `pylearn2.termination_criteria` module, as well as the `termination_criterion` of `SGD` and maybe other training algorithms or objects, and replace everything in the tutorials and documentation.

**Comment by goodfeli** [comment]
Please don't make this change.

"Termination criterion" is an actual computer science term in English. If you do a Google search for "termination criterion" it gets over 100,000 hits with many of them being computer science papers.

If you do a Google search for "continuation criterion" it says "Did you mean: "continuity criterion""

The top hit has nothing to do with terminating a while loop: http://arxiv.org/abs/1406.0165

**Comment by lamblin** [comment]
So, how do you propose we handle the current situation, where "and" and "or" mean the exact opposite of their meaning in English, without breaking backward compatibility?

**Comment by dwf** [comment]
Idea: deprecate And/Or, go with Conjunction/Disjunction with correct
meaning.

**Comment by lamblin** [comment]
Right. Since there can be more than one criterion, we could also use "Any" and "All", which are almost as short as the original.

**Comment by hemanthsavasere** [comment]
Hello,
Is this bug fixed yet because I want to work on it.

**Comment by nouiz** [comment]
Pylearn2 is dead. Nobody will review the PR. There is other framework on
top of Theano like Lasagne, keras or block that are still alive and would
welcome contribution.

Le lun. 15 mai 2017 20:09, Hemanth Savasere <notifications@github.com> a
écrit :
